### ISSUE TITLE
Optimal Config: uncheck FX Browser, Options menu, Show in FX list, Plug-in type prefixes

### ISSUE THREAD

**Issue by ScottChesworth** [jcsteh/osara#1153]
This is checked by default, but with it unchecked, browsing through FX becomes waaaaay more productive as REAPER strips the type prefixed before each entry in the list control. I demoed the difference during a stream and received enthusiastic requests for unchecking this to be added to our optimal config.

In a futile attempt to implement it, I saved two copies of an ini file with the state of this setting being the only variable.
When checked (default), this line is in reaper.ini:
uiflags=4
When unchecked, that becomes:
uiflags=16777220
And when I read that, my brain did:
sleep=1000000000000000

Is this one of those situations where Cockos are storing multiple things in that key? I feel like something similar to this came up in a previous addition to optimal config and I think @jcsteh knew what was going on then. Just between us, I only pretended to understand last time around.

HELP!

**Comment by jcsteh** [comment]
Based on the values you provided, assuming this was in the [REAPER] section of the ini file, you want this:

`{"REAPER", "uiflags", 1 << 24, 16777220},`

You should definitely test this though. :)

**Comment by jcsteh** [comment]
> Is this one of those situations where Cockos are storing multiple things in that key?

Yes. It looks like they're storing at least 24 flags (likely on/off settings) in that single key. 😲

**Comment by ScottChesworth** [comment]
Just tried the suggestion and it didn't work unfortunately. OSARA builds, the prompt appears, but the setting isn't adjusted when I restart REAPER. Same behaviour using my ini and a fresh portable.
Any advice on how to troubleshoot is very welcome.

**Comment by jcsteh** [comment]
> assuming this was in the [REAPER] section of the ini file

It isn't. It's in the [REAPER-fxadd] section of the ini file. Sorry, I intended my "assuming" comment to mean "please check this before assuming I'm correct", but it probably wasn't clear. :)

So, that means the code should be:

`{"REAPER-fxadd", "uiflags", 1 << 24, 16777220},`

However, when I looked locally, unchecking this option gave me 16777216, not 16777220. It could be that I changed some other setting at some point, but just to be sure, you may wish to check the value with a clean REAPER config. If it ends up being 16777216, you want this line:

`{"REAPER-fxadd", "uiflags", 1 << 24, 16777216},`
